### PR TITLE
com_fields view review

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

Review frontend com_fields layouts

### Testing Instructions

Make sure fields are still rendern

### Expected result

Works

### Actual result

Works, with smal cs issues

### Documentation Changes Required

none

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 1735721076a02eb54cea11101ed59bf01a29ecc1<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17231">issues.joomla.org/tracker/joomla-cms/17231</a>.</sub>

File: components/com_fields/layouts/fields/render.php [hunk 1]
```diff
@@ -49,15 +49,16 @@
...
+		<?php // If the value is empty do nothing ?>
+		<?php if (!isset($field->value) or $field->value == '') : ?>
+			<?php continue; ?>
+		<?php endif; ?>
+		<?php $class = $field->params->get('render_class'); ?>
+		<dd class="field-entry <?php echo $class; ?>">
+			<?php echo FieldsHelper::render($context, 'field.render', array('field' => $field)); ?>
+		</dd>
+	<?php endforeach; ?>
 </dl>
```
Comment (laoneo) [inline components/com_fields/layouts/fields/render.php]: Would make sense to add a new line here as well if you do some CS cleanup in this file.

File: components/com_fields/layouts/field/render.php [hunk 2]
```diff
@@ -23,6 +23,7 @@
 	return;
 }
 ?>
+
```
Comment (laoneo) [inline components/com_fields/layouts/field/render.php]: This adds an empty line in the generated output, is this intended?

File: components/com_fields/layouts/fields/render.php [hunk 3]
```diff
@@ -49,15 +49,16 @@
 	return;
 }
 ?>
+
```
Comment (laoneo) [inline components/com_fields/layouts/fields/render.php]: This adds an empty line in the generated output, is this intended?

File: components/com_fields/layouts/fields/render.php (hunk 1 above)
Comment (zero-24) [reply components/com_fields/layouts/fields/render.php]: done thanks.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on e1974b3135c62b03b6e1a6af12e91fc460c5b298<br /><br />Code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17231">issues.joomla.org/tracker/joomla-cms/17231</a>.</sub>

File: components/com_fields/layouts/fields/render.php [hunk 4]
```diff
@@ -48,16 +48,17 @@
...
-	<?php $class = $field->params->get('render_class'); ?>
-	<dd class="field-entry <?php echo $class; ?>">
-		<?php echo FieldsHelper::render($context, 'field.render', array('field' => $field)); ?>
-	</dd>
-<?php endforeach; ?>
-</dl>
\ No newline at end of file
+	<?php foreach ($fields as $field) : ?>
+		<?php // If the value is empty do nothing ?>
+		<?php if (!isset($field->value) or $field->value == '') : ?>
```
Comment (Quy) [inline components/com_fields/layouts/fields/render.php]: Do you want to change `or` to `||`?

**Comment by ghost** [comment]
@brianteeman can you please test again?

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on fbf91e0d241d28a5625c5072ac6832a50a491ea8<br /><br />Code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17231">issues.joomla.org/tracker/joomla-cms/17231</a>.</sub>

**Comment by n9iels** [comment]
I have tested this item :white_check_mark: successfully on 35b0b411d7f301c877e314208906863a96560e71<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17231">issues.joomla.org/tracker/joomla-cms/17231</a>.</sub>

**Comment by ghost** [comment]
@Quy can you please retest?

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 35b0b411d7f301c877e314208906863a96560e71<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17231">issues.joomla.org/tracker/joomla-cms/17231</a>.</sub>

**Comment by ghost** [comment]
RTC after two successful tests.

**Pull request merged**